srcpkgs/XMousePasteBlock/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'XMousePasteBlock'
+pkgname=XMousePasteBlock
+version=1.3
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="libev-devel libX11-devel libXi-devel base-devel"
```

Comment:
Curious, why is `base-devel` here? it's not required to build using `./xbps-src pkg XMousePasteBlock` for native (neither `glibc` or `musl`) or cross-builds

Reply:
Good catch. I don't remember why I had that there, but It's removed now. Thanks for letting me know.